src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -10,13 +10,22 @@ namespace Categories
     using Tests.Permutations;
     using Tests.Tools;
     using Variables;
-    using VisualStudioDebugHelper;
 
     public abstract class Base
     {
         public static string SessionId;
-        static readonly bool InvokeEnabled = bool.Parse(ConfigurationManager.AppSettings["InvokeEnabled"]);
-        static readonly TimeSpan MaxDuration = TimeSpan.Parse(ConfigurationManager.AppSettings["MaxDuration"]);
+        static readonly bool InvokeEnabled;
+        static readonly TimeSpan MaxDuration;
+
+        static Base()
+        {
+            // we need to get the correct config file, as .net core will try to load nunits test harness configuration file
+            // see https://github.com/dotnet/corefx/issues/22101
+            var assemblyLocation = Assembly.GetExecutingAssembly().Location;
```

Comment:
I was wondering if it wouldn't be simpler to replace the config file with a code first approach. @ramonsmits how often are these values changed?

Reply:
I'm fine with doing a code only approach for most values, especially these two. They could just be static readonly.

Reply:
Well, `InvokeEnabled` was used by the buildserver. Just make it a code var. Build would need to be addressed anyway as it is currently broken.